runtime/compiler/z/codegen/J9TreeEvaluator.cpp

```diff
@@ -12136,6 +12148,7 @@ J9::Z::TreeEvaluator::tstartEvaluator(TR::Node * node, TR::CodeGenerator * cg)
    //    POST Deps
 
    TR::Compilation *comp = cg->comp();
+   TR_J9VMBase *fej9 = (TR_J9VMBase *)(cg->fe());
```

Comment:
Let's try an avoid introducing new unsafe C-style casts into the codebase and use `static_cast<TR_J9VMBase*>(cg->fe())` here instead.